spring-kafka/src/main/java/org/springframework/kafka/listener/adapter/DelegatingInvocableHandler.java

```diff
@@ -114,13 +129,35 @@ public DelegatingInvocableHandler(List<InvocableHandlerMethod> handlers,
 	 * @param beanExpressionResolver the resolver.
 	 * @param beanExpressionContext the context.
 	 * @param beanFactory the bean factory.
-	 * @since 2.1.11
+	 * @deprecated in favor of
+	 * {@link #DelegatingInvocableHandler(List, InvocableHandlerMethod, Object, BeanExpressionResolver, BeanExpressionContext, BeanFactory, Validator)}
+	 * @since 2.5.11
 	 */
+	@Deprecated
 	public DelegatingInvocableHandler(List<InvocableHandlerMethod> handlers,
 			@Nullable InvocableHandlerMethod defaultHandler,
 			Object bean, BeanExpressionResolver beanExpressionResolver, BeanExpressionContext beanExpressionContext,
 			@Nullable BeanFactory beanFactory) {
 
+		this(handlers, defaultHandler, bean, beanExpressionResolver, beanExpressionContext, beanFactory, null);
+	}
+
+	/**
+	 * Construct an instance with the supplied handlers for the bean.
+	 * @param handlers the handlers.
+	 * @param defaultHandler the default handler.
+	 * @param bean the bean.
+	 * @param beanExpressionResolver the resolver.
+	 * @param beanExpressionContext the context.
+	 * @param beanFactory the bean factory.
+	 * @param validator the validator.
+	 * @since 2.1.11
```

Comment:
`2.5.11`?

Why then you say in the PR description that we need to back-port it back to `2.3.x`?

Reply:
Fixed and updated PR comment.